rust-toolchain.toml

```diff
@@ -1,9 +1,11 @@
 [toolchain]
+profile = "default"
 channel = "beta"
 targets = [
+    "aarch64-apple-darwin",
+    "x86_64-apple-darwin",
+    "x86_64-pc-windows-msvc",
     "x86_64-unknown-linux-musl",  # Tools
```

Comment:
Will adding these extra targets to CLI mean that it'll also attempt to build / test for those if we pass `--all-targets`?

Reply:
I guess so. I haven't noticed a big increase in CI runtime in this branch so it seems to be fine. On the other hand I'm not sure this is used in CI, since we only get the channel from this file in `init_rust_job`. Locally I usually only deal with the target for my dev machine and wasm32-unknown-unknown, the rest would be for cross compilation, right? Happy to change this as well.

Reply:
As long as when running Clippy etc. we don't get errors in the CLI for wasm32-unknown-unknown it should be fine